app/src/reducers/notifyReducer.js

```diff
@@ -0,0 +1,16 @@
+import initialState from './initialState';
+import {SHOW_NOTIFY} from '../actions/actionTypes';
+
+export default function notify(state = initialState.notify, action) {
+  let newState;
+  switch (action.type) {
+    case SHOW_NOTIFY:
+      newState = {
+        msg: action.msg,
+        timestamp: +new Date()
```

Comment:
may be good to end-of-line comment this ex:
`//create instantaneous JS timestamp`